operations/template/alert.tf

```
@@ -120,3 +120,45 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "database_token_expired_a
     ]
   }
 }
+resource "azurerm_monitor_metric_alert" "low_instance_count_alert" {
+  count               = local.non_pr_environment ? 1 : 0
+  name                = "cdcti-${var.environment}-azure-low-instance-count-alert"
+  resource_group_name = data.azurerm_resource_group.group.name
+  scopes              = [azurerm_monitor_autoscale_setting.api_autoscale.id]
+  description         = "The instance count in ${var.environment} is too low"
+  severity            = 2       // warning
+  frequency           = "PT1M"  // Checks every 1 minute
+  window_size         = "PT15M" // Every Check, looks back 15 minutes in history
+
+  criteria {
+    metric_namespace = "Microsoft.Insights/autoscalesettings"
+    metric_name      = "ObservedCapacity"
+    aggregation      = "Average"
+    operator         = "LessThanOrEqual"
+    // This threshold is based on the autoscale settings in app.tf
+    // How should we tune these numbers if we've scaled up higher than the initial count of 3/1?
+    threshold = local.higher_environment_level ? 2.5 : 0.5
```

We might be able to do something like this...

```suggestion
    threshold = azurerm_monitor_autoscale_setting.api_autoscale.profile.capacity.default - 0.5
```

Or we might need to do something like this...

```suggestion
    threshold = azurerm_monitor_autoscale_setting.api_autoscale.profile[0].capacity.default - 0.5
```